mod/node-core/pkg/components/interfaces.go

```
@@ -1162,6 +1168,10 @@ type (
 		RandaoAtEpoch(slot math.Slot, epoch math.Epoch) (common.Bytes32, error)
 	}
 
+	BlobBackend[BeaconBlockHeaderT any] interface {
+		BlobSidecarsAtSlot(slot math.Slot, indices []uint64) ([]*types.BlobSidecarData[BeaconBlockHeaderT], error)
```

golangci-lint

the line is 108 characters long, which exceeds the maximum of 80 characters.

(lll)